packages/aws-cdk-lib/aws-ec2/lib/vpc-flow-logs.ts

```diff
@@ -688,6 +711,11 @@ export interface FlowLogOptions {
    * The maximum interval of time during which a flow of packets is captured
    * and aggregated into a flow log record.
    *
+   * When creating flow logs for a Transit Gateway or Transit Gateway Attachment,
+   * it is not possible to specify TEN_MINUTES.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-flowlog.html#cfn-ec2-flowlog-maxaggregationinterval
+   *
    * @default FlowLogMaxAggregationInterval.TEN_MINUTES
```

Comment:
I changed the description and the default.

```suggestion
   * When creating flow logs for a Transit Gateway or Transit Gateway Attachment,
   * this property must be ONE_MINUTES.
   *
   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-flowlog.html#cfn-ec2-flowlog-maxaggregationinterval
   *
   * @default - FlowLogMaxAggregationInterval.ONE_MINUTES if Transit Gateway or
   * Transit Gateway Attachment, otherwise FlowLogMaxAggregationInterval.TEN_MINUTES.
```